static-analysis/teammates-eslint-es6.yml

```diff
@@ -20,6 +20,15 @@ rules:
     # enforce a maximum line length
     max-len: ["error", 125]
 
+    # disallow continue statements
```

Comment:
Oh this was copied from the original ESLint config 😅
There the usage was the rule description, as taken from [the official page](http://eslint.org/docs/rules/), on top of the rule name.